tests/manual/token.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * @license Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals console, window, document, XMLHttpRequest */
+
+import ClassicEditor from '@ckeditor/ckeditor5-editor-classic/src/classiceditor';
+import ArticlePluginSet from '@ckeditor/ckeditor5-core/tests/_utils/articlepluginset';
+import CloudServices from '../../src/cloudservices';
+
+import { TOKEN_URL, UPLOAD_URL } from '../_utils/cloud-services-config';
+
+const output = document.querySelector( '#output' );
+const refreshTokenButton = document.querySelector( '#refresh-token' );
+
+ClassicEditor
+	.create( document.querySelector( '#editor' ), {
+		cloudServices: {
+			tokenUrl: getToken,
+			uploadUrl: UPLOAD_URL
+		},
+		plugins: [ ArticlePluginSet, CloudServices ],
+		toolbar: [ 'heading', '|', 'undo', 'redo' ]
+	} )
+	.then( editor => {
+		window.editor = editor;
+
+		const cloudServices = editor.plugins.get( CloudServices );
+
+		refreshTokenButton.addEventListener( 'click', () => {
+			cloudServices.token._refreshToken();
```

Comment:
I am against using protected API in the manual test. One can refresh the page to test it again.